bin/PurgeBadJobs.py

```diff
@@ -0,0 +1,127 @@
+#!/miniconda-latest/bin/python3
```

Comment:
Because the kbase and root have different envs, and this forces you to use the python that is installed in that dir

Reply:
> Because the kbase and root have different envs

I'm not sure I understand this - kbase and root are different users?

I'm having heartburn here because now this script isn't portable - if someone tries to run it on login1 or something to clean up a DB before starting the server it won't work (at least calling it as is). The standard shebang is portable.

What if we kept the script as portable and in the crontab invocation specified `/miniconda... PurgeBadJobs.py` there since it's the crontab that needs the specific environment?

Reply:
![image](https://user-images.githubusercontent.com/1258634/128460849-4db521fa-2daf-4df1-a6cd-4d08e503dd0b.png)

Missed a word, meant the kbase and root users have different envs.

This shebang is not really what is making this script not portable. A bunch of other stuff does though like
* Requirement of an undocumented config file, which has undocumented required parameters, 
* No command line options to pass into this script
* No documentation on how the script runs or why it should run in the script, or why we need this script
* Reliance of ee2 functions without any versioning for their functionality

I didn't really imagine this script to be portable, or a need for running this on other machines, or needing something to clean up the DB before starting up the server. 
I would probably just write a one time mongo query if wanted to clean things up if that were the case, and I wouldn't bother notifying slack.

Reply:
Actually, one benefit of having that shebang is it lets you know the python path of the python with ee2 installed.
Since you insist on changing this shebang, I will do it.

Reply:
Those are all usability issues rather than [portability](https://en.wikipedia.org/wiki/Software_portability) issues in the sense that I'm using them

Reply:
Anyway :+1: